Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -81,35 +81,44 @@ During this stage, the **release manager** has the following tasks:
 
+This process is now automated using the `submit_for_review` lane. Just trigger the lane from the `#ios-build` channel using `/stevenson fastlane submit_for_review target:<target> build:<testflight_build>`.
+
+<details><summary>If for some reason you need to do this process manually instead of using the lane, you can follow these manual steps</summary>
+	
 1. Make sure *Manually release this version* is selected in `Version Release`.
 2. Select *Use phased release*
 3. When submitting to release, you are asked if the app uses the Advertising Identifier (IDFA). The answer is YES. You are then presented with three options please select as followed:
 	1. 🚫 Serve advertisements within the app
 	2. ✅ Attribute this app installation to a previously served advertisement
 	3. ✅ Attribute an action taken within this app to a previously served advertisement
+</details>
 
 ### Phase 6: Closure
 *It starts after the app is accepted by Apple and final internal approval*
 
-1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
-1. Press `Release this version` in App Store Connect
-1. Create a tag named `{appname}/{version}` (e.g. `babylon/4.1.0`) on the release commit and create a GitHub release for that new tag
-   * Make sure you create separate tags (and GitHub releases) for each app released on the AppStore (eg. Babylon 4.1.0 and Telus 4.1.0 would each have their own `babylon/4.1.0` and `telus/4.1.0` tags)
-   * Set the body of the GitHub release to the content of the Release Notes for the app
-   * Attach the zipped `xcarchive` as an artefact to the GitHub release (if you're using the automated release command, you can find the `*.xcarchive.zip` in the Artifacts top section in the CI build).
+Most of the steps for this phase have been automated using the `finish_release` lane.
+
+1. Trigger the lane from the `#ios-build` channel using `/stevenson fastlane finish_release target:<target>`.
+
+    <details><summary>If for some reason you need to do this process manually instead of using the lane, you can follow these manual steps</summary>
+
+    1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
+    1. Press `Release this version` in App Store Connect
+    1. Create a tag named `{appname}/{version}` (e.g. `babylon/4.1.0`) on the release commit and create a GitHub release for that new tag
+       * Make sure you create separate tags (and GitHub releases) for each app released on the AppStore (eg. Babylon 4.1.0 and Telus 4.1.0 would each have their own `babylon/4.1.0` and `telus/4.1.0` tags)
+       * Set the body of the GitHub release to the content of the Release Notes for the app
+       * Attach the zipped `xcarchive` as an artefact to the GitHub release (if you're using the automated release command, you can find the `*.xcarchive.zip` in the Artifacts top section in the CI build).
+   
+    </details>
+
 1. Merge `release` branch back to `develop`:
```

Comment:
Note: this is planned to be automated in the future too and be integral part of the `finish_release` lane, it's just not implemented yet and will only be added to the lane in the future when I have some more bandwidth. This is why it's still listed as manual instruction here.